diff --git a/modules/custom/activity_creator/activity_creator.post_update.php b/modules/custom/activity_creator/activity_creator.post_update.php
index 90fb27cf7..bb143a0b2 100644
--- a/modules/custom/activity_creator/activity_creator.post_update.php
+++ b/modules/custom/activity_creator/activity_creator.post_update.php
@@ -109,7 +109,9 @@ function activity_creator_post_update_8802_remove_orphaned_activities(&$sandbox)
     // If 'count' is empty, we have nothing to process.
     if (empty($sandbox['count'])) {
       $sandbox['#finished'] = 1;
-      drush_print("No entities to be processed.");
+      if (function_exists('drush_print')) {
+        drush_print("No entities to be processed.");
+      }
       return;
     }
 
@@ -144,7 +146,9 @@ function activity_creator_post_update_8802_remove_orphaned_activities(&$sandbox)
 
   // While processing our batch requests, we can send a helpful message
   // to the command line, so developers can track the batch progress.
-  drush_print('Progress: ' . (round($progress_fraction * 100)) . '% (' . $sandbox['progress'] . ' of ' . $sandbox['count'] . ' activities processed)');
+  if (function_exists('drush_print')) {
+    drush_print('Progress: ' . (round($progress_fraction * 100)) . '% (' . $sandbox['progress'] . ' of ' . $sandbox['count'] . ' activities processed)');
+  }
 
   // Drupal’s Batch API will stop executing our update hook as soon as
   // $sandbox['#finished'] == 1 (viz., it evaluates to TRUE).
@@ -186,7 +190,9 @@ function activity_creator_post_update_8803_remove_activities_with_no_related_ent
     // If 'count' is empty, we have nothing to process.
     if (empty($sandbox['count'])) {
       $sandbox['#finished'] = 1;
-      drush_print("No entities to be processed.");
+      if (function_exists('drush_print')) {
+        drush_print("No entities to be processed.");
+      }
       return;
     }
 
@@ -223,7 +229,9 @@ function activity_creator_post_update_8803_remove_activities_with_no_related_ent
   $progress_fraction = $sandbox['progress'] / $sandbox['count'];
 
   // Send a helpful message to the command line.
-  drush_print('Progress: ' . (round($progress_fraction * 100)) . '% (' . $sandbox['progress'] . ' of ' . $sandbox['count'] . ' activities processed)');
+  if (function_exists('drush_print')) {
+    drush_print('Progress: ' . (round($progress_fraction * 100)) . '% (' . $sandbox['progress'] . ' of ' . $sandbox['count'] . ' activities processed)');
+  }
 
   // Tell the Batch API about status of this process.
   $sandbox['#finished'] = empty($sandbox['activities_id']) ? 1 : ($sandbox['count'] - count($sandbox['activities_id'])) / $sandbox['count'];
